Doc/whatsnew/3.14.rst

```diff
@@ -438,6 +438,15 @@ sys
   which only exists in specialized builds of Python, may now return objects
   from other interpreters than the one it's called in.
 
+turtle
+------
+
+* Add context managers for :func:`turtle.fill`, :func:`turtle.poly` and
+  :func:`turtle.no_animation` for
+  :func:`turtle.begin_fill`/:func:`turtle.end_fill`,
+  :func:`turtle.begin_poly`/:func:`turtle.end_poly`, and
+  :func:`turtle.tracer(0)`/:func:`turtle.tracer(1)`, respectively.
```

Comment:
I don't think we need that much of details. Saying which context managers were added should be sufficient or we should use a bullet point list (otherwise it's a bit compact toread).

For the docs failure, you can use 

```rst
:func:`turtle.tracer(0) <turtle.tracer>`
```

It will render `turtle.tracer(0)` with a link to `turtle.tracer`.